webrtc/_compositions/go2go.toml

```diff
@@ -0,0 +1,64 @@
+[metadata]
+  name = "go2go {{ .Env.GitReference }}"
+
+[global]
+  plan = "libp2p/webrtc"
+  case = "webrtc"
+  builder = "docker:go"
+  runner = "local:docker"
+  total_instances = 2
+
+  [global.build_config]
+    enable_go_build_cache  = false # see https://github.com/testground/testground/issues/1361
+    go_proxy_mode          = "remote"
+    go_proxy_url           = "https://proxy.golang.org"
+
+[[groups]]
+  id = "server"
+  instances = { count = 1 }
+  builder = "docker:go" # override builder here
+
+  [groups.build_config]
+    path = "./go"
+    build_base_image = 'golang:1.19-buster'
+    build_args = { image = "golang:1.19-bullseye" }
+    modfile = "go.orig.mod"
+
+  [groups.run.test_params]
+    expected_implementation = "go"
```

Comment:
this parameter was copied over, no? But not really that relevant for your test I think?

Reply:
Probably not relevant, I'd guess. But I'm not really sure, to be honest.

Reply:
These parameters are things you use to pass info at start-up time to your test, in order to perhaps configure something. So if you are not doing something with the `expected_implementation` param, you do not need it. These variables are in the go sdk accessed using the methods from https://pkg.go.dev/github.com/testground/sdk-go@v0.3.0/runtime#RunParams ( e.g. `StringParam`)